static/app/views/alerts/list/rules/row.tsx

```diff
@@ -496,8 +496,8 @@ const StyledChevron = styled(IconChevron)`
 `;
 
 const StyledIconUser = styled(IconUser)`
-  /* We need this to center with Avatar */
-  margin-right: 2px;
+  /* This is used to make the default icon the same size */
```

Comment:
When the team icon is used instead of `Unassigned`, I noticed that the icon touches the dropdown (before these changes and after). Is this the expected styling, or should there be a gap?

![Screenshot 2024-01-25 at 2 02 51 PM](https://github.com/getsentry/sentry/assets/22582037/f4ee68ff-54bf-4974-be7d-bab0de5c98f5)

Reply:
Not sure.. but i like that idea! i'll add it to both icons to give it a little more breathing room.

Reply:
updated so it matches the unassigned spacing:
![Screenshot 2024-01-25 at 3 11 09 PM](https://github.com/getsentry/sentry/assets/1569818/b3d7e82e-6e43-4a50-b6ca-8eec3b3bf7e2)